src/Stratis.Features.FederatedPeg/PartialTransactionsBehavior.cs

```diff
@@ -99,10 +99,22 @@ private async Task OnMessageReceivedAsync(INetworkPeer peer, IncomingMessage mes
                 return;
             }
 
+            if (transfer[0].PartialTransaction == null)
+            {
+                this.logger.LogTrace("OnMessageReceivedAsync: Deposit {0}, PartialTransaction not found.", payload.DepositId);
```

Comment:
Perhaps replace OnMessageReceivedAsync with nameof()? i.e.

this.logger.LogTrace("{0}: Deposit {1}, PartialTransaction not found.", payload.DepositId, nameof(this.OnMessageReceivedAsync));

Reply:
While agree this is the same as the  other logs in the method so its consistent, not sure its worth changing to wait for a build again.

Reply:
@dangershony are you going to make these changes?

Reply:
Only on the new methods or on all the logs in the class?
I am saying this is consistent so its fine like this.

Reply:
ok this is now fixed